pkg/manager/install.go

```diff
@@ -0,0 +1,145 @@
+package manager
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/goods"
+	"github.com/replicatedhq/embedded-cluster/pkg/helpers"
+	"github.com/replicatedhq/embedded-cluster/pkg/helpers/systemd"
+	"github.com/replicatedhq/embedded-cluster/pkg/runtimeconfig"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	managerDropInFileContents = `[Service]
+# Empty ExecStart= will clear out the previous ExecStart value
+ExecStart=
+ExecStart=%s start
+`
+)
+
+type LogFunc func(string, ...interface{})
+
+// UnitName returns the name of the systemd unit for the manager service.
+func UnitName() string {
+	return fmt.Sprintf("%s.service", runtimeconfig.ManagerServiceName)
```

Comment:
would this actually match the systemd unit file name used on initial install? the `ManagerServiceName` variable resolves to the executable file name, which won't match the installer binary name used to set up the cluster.

Reply:
fixed